### PR TITLE
css for blog module; meta facebook et title, indent

### PR DESCRIPTION
**Pull request by tomplays**

### CODE REVIEW

File: wp-content/themes/nuitdeboo-child/templates/module-participate.php [hunk 1]
```diff
@@ -35,7 +28,7 @@ function getAttachmentThumb($id) {
 		<?php foreach ($pages_sub as $p) : ?>
 			<div class="participate-action">
 				<div class="participate-action__image">
-					<img class="img-responsive" src="<?php echo getAttachmentThumb($p->ID) ?>">
+					<img class="img-responsive" src="<?php echo getAttachmentThumb($p->ID); ?>">
```
Comment (alexsegura) [inline wp-content/themes/nuitdeboo-child/templates/module-participate.php:31]: FYI semi-colons are useless for one-liners

Comment (tomplays) [inline wp-content/themes/nuitdeboo-child/templates/module-participate.php:31]: ok ! :)

File: wp-content/themes/nuitdeboo-child/templates/head.php [hunk 2]
```diff
@@ -2,15 +2,30 @@
   <meta charset="utf-8">
   <meta http-equiv="x-ua-compatible" content="ie=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1">
-  <?php wp_head(); ?>
-
-
+  	  <?php wp_head(); ?>
 	  <link rel="shortcut icon" href="<?php echo get_stylesheet_directory_uri(); ?>/dist/images/favicon.ico" />
-	  <meta property="og:locale" content="fr_FR"> <!-- a tester : <?php bloginfo( 'language' ); ?> !-->
+	  <meta name="google-site-verification" content="_83yK55ZXdmMvZ2KB8Yn-lp8zNuoYT6IbX2iT-oXqo0" />
```
Comment (alexsegura) [inline wp-content/themes/nuitdeboo-child/templates/head.php]: There is already a file in the docroot, not really useful

File: wp-content/themes/nuitdeboo-child/templates/module-facebook-news.php [hunk 3]
```diff
@@ -0,0 +1,19 @@
+<?php
+echo '<div id="fbnews" class="section home-widget-container">';
```
Comment (alexsegura) [inline wp-content/themes/nuitdeboo-child/templates/module-facebook-news.php]: Why use `echo` when you can write HTML? It would be more readable.

**Pull request merged**